src/main/java/step4/domain/RacingGame.java

```diff
@@ -0,0 +1,60 @@
+package step4.domain;
+
+import step4.utils.RandomNumGenerator;
+
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class RacingGame {
+    private final List<Car> cars;
+    private int tryNo;
+
+    public RacingGame(String carNames) {
+        this.cars = initCars(carNames);
+    }
+    public RacingGame(String carNames, int tryNo) {
+        this.cars = initCars(carNames);
+        this.tryNo = tryNo;
+    }
+
+    private static List<Car> initCars(String carNames) {
```

Comment:
멤버 메서드로 사용 해도 문제 없네요!
```suggestion
    private List<Car> initCars(String carNames) {
```